query/src/test/java/org/infinispan/query/distributed/MassIndexingTest.java

```diff
@@ -73,7 +73,7 @@ public void testOverlappingMassIndexers() {
 
       latch.countDown();
 
-      assertTrue(isSuccess(first) && isError(second) || isError(first) && isSuccess(second));
+      assertTrue(isSuccess(second) && isError(first) || isSuccess(first) && isError(second));
```

Comment:
This change is no-op @gustavonalle . Are you sure you actually wrote what you intended?

Reply:
This will reproduce the failure much easier, as there is only a small probability that the first indexer will not be executed